test/value.cpp

```diff
@@ -499,6 +499,13 @@ class value_test
                 BOOST_TEST(jv.is_object());
                 BOOST_TEST(*jv.storage() == *sp);
             }
+            {
+                // verify moved from
+                object obj{{"a", 1}, {"a", 1}};
+                BOOST_TEST(obj.capacity());
```

Comment:
I would not depend on conversion to bool. In other words, spell this out: `obj.capacity() > 0`